otherlibs/dune-rpc/dune_rpc.mli

```diff
@@ -86,6 +86,80 @@ module V1 : sig
     val to_string_absolute : t -> string
   end
 
+  module Ansi_color : sig
+    module RGB8 : sig
+      type t
+    end
+
+    module RGB24 : sig
+      type t
+    end
```

Comment:
I'm not sure if we want to add any functions allowing us to inspect the contents. For now they are a black box to the consumer of RPC.

Reply:
How are clients supposed to use them then?

Reply:
I'll add some functions.